docs/assets/polished.js

```diff
@@ -1042,7 +1042,7 @@ var functionsMap = {
 /** */
 
 /**
- * String to represent common easing functions as demonstrated here: (github.com/jaukia/easie).
+ * String to represent commong easing functions as demonstrated here: (github.com/jaukia/easie).
```

Comment:
Odd that this is here, but not in my actual changes? Maybe this hasn't been built in awhile?